appcenter-data/android/build.gradle

```diff
@@ -24,8 +24,11 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'
-    api 'com.microsoft.appcenter:appcenter-data:2.1.0-7+df4dfd7f7'
+    api 'com.microsoft.appcenter:appcenter-data:2.1.0'
 
-    //api project(':AppCenterReactNativeShared')   // For testing with TestApp
-    api 'com.microsoft.appcenter.reactnative:appcenter-react-native:2.0.0'
+    api project(':AppCenterReactNativeShared')   // For testing with TestApp
+//    api 'com.microsoft.appcenter.reactnative:appcenter-react-native:2.1.0'
+
+//    TODO Remove gson once Android SDK dependencies are updated
+    implementation 'com.google.code.gson:gson:2.8.5'
```

Comment:
@guperrot @dhei I added a TODO to remove this after the Android SDK has been updated.